instrumentation/net/http/otelhttp/common.go

```diff
@@ -37,6 +37,13 @@ const (
 	ServerLatency         = "http.server.duration"                // Incoming end to end duration, microseconds
 )
 
+// Client HTTP metrics.
+const (
+	ClientRequestContentLength  = "http.client.request_content_length"  // Outgoing request bytes total
+	ClientResponseContentLength = "http.client.response_content_length" // Outgoing response bytes total
```

Comment:
```suggestion
	ClientRequestContentLength  = "http.client.request.size"  // Outgoing request bytes total
	ClientResponseContentLength = "http.client.response.size" // Outgoing response bytes total
```

per https://github.com/open-telemetry/opentelemetry-specification/blob/v1.17.0/specification/metrics/semantic_conventions/http-metrics.md#http-client